app/src/pages/inside/logsPage/logItemInfo/logItemInfo.jsx

```diff
@@ -206,6 +213,24 @@ export class LogItemInfo extends Component {
     });
   };
 
+  handleUnlinkTicket = (logItem) => (ticketId) => {
```

Comment:
You don't need to create a function that returns another function.
You already have access to `logItem` from `props`.

Reply:
Fixed.